src/app/course-home-page/all-links/all-links.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import { Http, Response } from '@angular/http';
+
+
+@Component({
+  selector: 'app-all-links',
+  templateUrl: './all-links.component.html',
+  styleUrls: ['./all-links.component.css']
+})
+export class AllLinksComponent implements OnInit {
+  contentData= [];
+  toggle= true;
+  constructor(private http: Http) { }
+
+    ngOnInit() {
+      this.getContents();
+    }
+    getData() {
+    return this.http.get('https://cdn.contentful.com/spaces/l99z1yyviltd/entries?access_token=5f91dcaac99947796b2f772dba878c5b58fab81d02375c7b9d3f26a9e3a0f45b&content_type=sitemap')
```

Comment:
don't add http calls directly from components. Use a service layer for this. @manikyak show him some example